### DISCUSSION TITLE
Supported macOS versions—apparently works on Catalina

### DISCUSSION THREAD

**Discussion by j13k** [podman-desktop/podman-desktop#464] [General]
I'm running Podman on a trusty old MacBook Pro which is stuck on macOS Catalina (10.15).

I was initially put off using Podman Desktop because the [Homebrew Cask recipe specifies Big Sur or greater](https://github.com/Homebrew/homebrew-cask/blob/8cc7abd1c1564752b84d57a9a88361677cddfd87/Casks/podman-desktop.rb#L20
) as a dependency:

```
  depends_on macos: ">= :big_sur"
```

Since then, I decided to try manually installing it from the release image, and to my delight, initial testing indicates it works as expected on Catalina.

I realise that the project team has to practically limit the environments it is able to support, but thought I'd mention this in case it helps others who want to use the software on older hardware.

**Comment by slemeur** [comment]
Thanks @j13k for sharing the information to the rest of the community !

**Comment by benoitf** [comment]
Hello @j13k I fixed the Podman Desktop formulae on HomeBrew
https://github.com/Homebrew/homebrew-cask/pull/132562

It is now requiring macOS Catalina instead of Big Sur
https://formulae.brew.sh/cask/podman-desktop
